dockstore_launcher_config/compose.config

```diff
@@ -39,16 +39,9 @@
 "GOOGLE_CLIENT_ID":"potato",
 "GOOGLE_CLIENT_SECRET":"potato",
 "HTTPS":false,
-"IS_FARGATE_DEPLOY":false,
```

Comment:
Note sure on these, but the `HTTPS`, `PUBLIC_LAUNCHER_IP_ADDRESS` may  be useless too. Do a search and see what uses them.

Reply:
Removed them